tensorflow/compiler/xla/python/pmap_lib.cc

```diff
@@ -382,7 +382,9 @@ xla::StatusOr<py::object> PmapFunction::Call(py::args args, py::kwargs kwargs) {
     arguments.signature.dynamic_arg_signatures.push_back(
         std::move(signature_or_error).ValueOrDie());
   }
-  arguments.signature.global_extra_jit_context = global_state.extra_jit_context;
+  CHECK(global_state.extra_jit_context.has_value());
```

Comment:
I think I'd avoid the CHECK here and just make the `arguments.signature.global_extra_jit_context` field optional as well.

Reply:
Good call, done.